src/modules/users/controllers/UserController.e2e.spec.ts

```diff
@@ -98,26 +112,20 @@ describe('UserController', () => {
 
       expect(response1.statusCode).toBe(200)
       expect(response2.statusCode).toBe(200)
-      expect(response1.json()).toEqual(response.json())
-      expect(response2.json()).toEqual(response.json())
+      expect(response1.json().data).toMatchObject(NEW_USER_FIXTURE)
+      expect(response2.json().data).toMatchObject(NEW_USER_FIXTURE)
     })
   })
 
   describe('DELETE /users/:userId', () => {
     it('resets cache after deletion', async () => {
       const token = await generateJwtToken(app.jwt, { userId: '1' }, 9999)
+      const newUser = await userRepository.createUser(NEW_USER_FIXTURE)
+      const { id } = newUser
 
-      const response = await injectPost(app, postCreateUserContract, {
```

Comment:
used the opportunity to clean up example tests, typically it only makes sense to send API requests for the part of the system under test, everything else can be manipulated directly inside DB.